tests/test_pfs.py

```diff
@@ -148,45 +148,10 @@ def test_put_file_bytes_bytestring():
     commit_infos = list(client.list_commit(repo_name))
     assert len(commit_infos) == 1
     assert commit_infos[0].commit.id == c.id
-    files = list(client.list_file('{}/{}'.format(repo_name, c.id), '.'))
+    files = list(client.list_file('{}/{}'.format(repo_name, c.id), ''))
```

Comment:
Not really relevant to this line, but we should change python-pachyderm to use the same short format as pachctl: `repo@commit[:file]` rather than `repo/commit`

Reply:
Nevermind, we already have https://github.com/pachyderm/python-pachyderm/issues/255. Marking resolved.